src/hooks/useBankingSlugs.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react'
+import { useClient, Q } from 'cozy-client'
+import { KONNECTOR_DOCTYPE } from 'doctypes'
+
+const useBankingSlugs = () => {
+  const client = useClient()
+  const [bankingSlugs, setBankingSlugs] = useState([])
+  const [isFetchingBankSlugs, setIsFetchingBankSlugs] = useState(true)
+
+  useEffect(() => {
+    const load = async () => {
+      const bankingKonnectors = await client.query(
+        Q(KONNECTOR_DOCTYPE).where({
+          categories: {
+            $elemMatch: {
+              $eq: 'banking'
+            }
+          }
+        })
+      )
```

Comment:
@paultranvan Is there any difference using elemMatch instead of just `categories: 'banking'` ? 

@doubleface Can you create an index please? Also, you should check the presence of hasMore because we can have pagination on that request.

Reply:
Since `categories` is an array, `categories: 'banking'` won't work. However, I think `categories: $in: ['banking']` is equivalent. But be aware that both $elemMatch and $in will perform a full-scan of the index.

Reply:
@paultranvan thanks. Didn't even think about categories being an array... 

@doubleface don't we have something else than this `categories` array to detect if we have this "banking information"?

Reply:
C'est la meilleure information pour représenter le fait que c'est un connecteur bancaire en tous cas. Toute autre information dans le manifest serait une déduction douteuse à mon avis. C'est requête est faite sur la liste des connecteurs installée ce qui ne constitue normalement pas un grand nombre.

Ok pour l'index et la pagination

Reply:
@Crash-- En fait, ici, puisque je ne suis pas dans un composant react, c'est un queryAll que je dois faire non ?

Reply:
Tu peux passer par un queryAll() effectivement.  

Que tu sois dans un composant ou non, le resultat de la query disposera toujours du hasMore et du fetchMore. Si tu regardes le code du queryAll() ça fait juste un while resp.hasMore() ;).